### PR TITLE
 fix: add Ethereum address field

### PR DESCRIPTION
**Pull request by anatollupacescu**
Add a new `migratePeer` method to update addressbook.

Add new field `ethereum address`

This change is [Reviewable](https://reviewable.io/reviews/ethersphere/bee/2126)

### CODE REVIEW

File: pkg/debugapi/peer.go [hunk 1]
```diff
@@ -63,13 +63,19 @@ func (s *Service) peerDisconnectHandler(w http.ResponseWriter, r *http.Request)
 	jsonhttp.OK(w, nil)
 }
 
+// Peer holds information about a Peer.
+type PeerDTO struct {
```
Comment (acud) [inline pkg/debugapi/peer.go:67]: DTO?

**Review by acud — APPROVED** [review]

**Review by mrekucci — APPROVED** [review]
:lgtm:

> Reviewed 5 of 10 files at r1, 5 of 5 files at r2.
> *[Reviewable](https://reviewable.io/reviews/ethersphere/bee/2126#-:-McTv48t4suGs9Qnb-q7:bnfp4nl)* status: all files reviewed, 2 unresolved discussions (waiting on @anatollupacescu and @esadakar)

___
*[pkg/settlement/swap/addressbook.go, line 71 at r2](https://reviewable.io/reviews/ethersphere/bee/2126#-McTu6BO4-I4dWQpzFqp:-McTu6BO4-I4dWQpzFqq:bbr5pyl) ([raw file](https://github.com/ethersphere/bee/blob/22f0a7c2a370e10a7df97675f61cfb235e483327/pkg/settlement/swap/addressbook.go#L71)):*
> ```Go
> 	}
> 
> 	cb, known, err := a.Chequebook(oldPeer)
> ```

This is first used before the `if know {` statement, can this be moved before that statement or do you need to check for the error before continuing with the next statements?

**Review by ralph-pichler — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/ethersphere/bee/2126)* status: all files reviewed, 2 unresolved discussions (waiting on @anatollupacescu, @esadakar, and @mrekucci)

___
*[pkg/settlement/swap/addressbook.go, line 71 at r2](https://reviewable.io/reviews/ethersphere/bee/2126#-McTu6BO4-I4dWQpzFqp:-McTvbSh5ROZ1xjai_wA:b-544uwa) ([raw file](https://github.com/ethersphere/bee/blob/22f0a7c2a370e10a7df97675f61cfb235e483327/pkg/settlement/swap/addressbook.go#L71)):*
<details><summary><i>Previously, mrekucci (Peter Mrekaj) wrote…</i></summary><blockquote>

This is first used before the `if know {` statement, can this be moved before that statement or do you need to check for the error before continuing with the next statements?
</blockquote></details>

the idea here is too make sure we can read all the required values before starting to write anything

**Review by mrekucci — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/ethersphere/bee/2126)* status: all files reviewed, 1 unresolved discussion (waiting on @anatollupacescu and @esadakar)

**Review by anatollupacescu — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/ethersphere/bee/2126)* status: all files reviewed, 1 unresolved discussion (waiting on @acud and @esadakar)

___
*[pkg/debugapi/peer.go, line 67 at r1](https://reviewable.io/reviews/ethersphere/bee/2126#-McPhq-k6DNdC5u4o9Pk-r1-67:-McU0uYt5gvZmGXMblaN:b-896fix) ([raw file](https://github.com/ethersphere/bee/blob/c4faba7735de3b24d0c60593134db8e312c85696/pkg/debugapi/peer.go#L67)):*
<details><summary><i>Previously, acud (acud) wrote…</i></summary><blockquote>

DTO?
</blockquote></details>

Done.

**Pull request merged**